src/util/getNotes.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { useAuth, useNote } from "../context";
+
+const getNotes = async ({ token, noteDispatch }) => {
+  try {
+    const url = "/api/notes";
+    const config = {
+      headers: {
+        authorization: token,
+      },
+    };
+    const response = await axios.get(url, config);
+    console.log("response get:", response);
+    if (response.status === 200) {
+      const { notes } = response.data;
+      noteDispatch({ type: "INITIALIZE", payload: { notes: notes } });
+    }
+  } catch (err) {
+    console.log(err);
```

Comment:
you can write "return err" instead of the console when you are done with testing.